alerts/system_alerts.libsonnet

```diff
@@ -12,7 +12,7 @@
               severity: 'warning',
             },
             annotations: {
-              message: '{{ $labels.node }} has been unready for more than an hour',
+              message: '{{ $labels.node }} has been unready for more than an hour.',
```

Comment:
semantically `has not been ready for X` can mean two things:
* "it is currently in the ready state but it has not been ready for long enough; as soon as it has been ready for X, this will go away"
* "it is in a `!ready` state for more than X"

I chose the `unready` wording so it would be clear that the alert refers to the latter.